New package: Turtles v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: Turtles
- Repository: https://github.com/gm-at-tau/Turtles.jl
- Created by: @gm-at-tau
- Version: v0.1.0
- Commit: 8112efc4ebac82de8e4b3499857a22d52f0854a5
- Reviewed by: @gm-at-tau
- Reference: https://github.com/gm-at-tau/Turtles.jl/issues/2
- Description: A code generation library
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598919e709ae77a3ad62ff5eadb373d9cb5d6dcd89ee52226b35364680ed7e1132f16bfead10b93f2b5f902f187e6ba2f421c64c59aa701ebbaa2548ba16072752cdb52ad30036f79b638a65c5e24b72b8cb3f4f63331920e3c9a604f9b1ebd148044cf871296971862854bd60d3527bd315d807ac3b4c6519c236d1266a030c6ff -->

goerz commented:
Thank you for submitting your package! Could you add a little bit of documentation before registering? [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a more detailed description of the package's purpose and a small usage example in the README. An important part of packages in General is that any potential user can figure out what the package is about and how to get started with using it. That is really difficult when there is no documentation.

In the longer term, I definitely recommend setting up a [Documenter](https://documenter.juliadocs.org/stable/)-based documentation. Before a `v1.0` release, or for smaller packages that can be effectively described entirely with their `README`, that's not a requirement, though.

gm-at-tau commented:
Thanks, I updated the README and will re-trigger the Registrator soon.
The package is documented with `@doc` as this is a smaller (< 1k lines ) package, but will consider Documenter on future versions.

JuliaRegistrator committed:
New package: Turtles v0.1.0

UUID: 37f3a7ca-22fe-4915-8a8b-6f135989f045
Repo: https://github.com/gm-at-tau/Turtles.jl.git
Tree: 8617218a3f4a1e2a80fa1a26f0106463577f15d4

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
